changelogs/unreleased/6202-repair-loop.yml

```
@@ -0,0 +1,6 @@
+description: Prevent repairs from restarting indefinitely.
```

If we include it in the changelog perhaps we should make it sound a bit less alarming by specifying when it triggers?
```suggestion
description: Prevent repairs from restarting indefinitely when a short deploy interval is set
```